Quadriga/src/main/java/edu/asu/spring/quadriga/dao/impl/UserDAO.java

```diff
@@ -114,6 +117,24 @@ public boolean addNewUserAccountRequest(String username, String password, String
             throw new QuadrigaStorageException(e);
         }
     }
+    
+    
+    public boolean addNewSocialUserAccountRequest(String username, String fullname, String email, String provider, String userIdOfProvider)  throws QuadrigaStorageException{
```

Comment:
Yes. Consolidated the two methods. Removed addNewSocialUserAccountRequest method.